sites/main-site/src/content/blog/2024/helpdesk.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: 'Office hours become Helpdesk'
+subtitle: 'A weekly drop-in sessions to connect and work together.'
+headerImage: 'https://images.unsplash.com/photo-1545239351-ef35f43d514b'
+headerImageAlt: 'Person using macbook pro and holding cappuccino'
+announcement:
+    text: 'Office hours become Helpdesk'
+    start: 2024-10-07T16:14:00+01:00
+    end: 2024-10-07T16:14:00+01:00
+pubDate: 2024-10-07T16:14:00+01:00
+authors:
+    - 'FriederikeHanssen'
+label:
+    - 'community'
+---
+
+# nf-core Helpdesk
+
+We are restarting nf-core office hours under a new name: *nf-core Helpdesk*.
+These are weekly drop-in sessions designed for nf-core users and developers seeking an opportunity to connect, chat, and work together.
+
+:::tip{.fa-hourglass-clock title="Too Long; Didn't Read"}
+
+**When**:
+
+-   Tuesdays, 2 PM Eastern Time
+-   Thursdays, 10 AM Central European Summer Time
+
+**Where**:
+
+- [Gather Town](https://nf-co.re/join#gather-town)
+- [Slack: #helpdesk]()
+
+:::
+
+During the summer we trialed the [nf-core office hours](https://nf-co.re/blog/2024/office_hours). After gathering feedback, we decided to continue with them under a new name and
+broader scope: **nf-core Helpdesk**
+
+In these sessions, some of the nf-core maintainer team and other seasoned community members will be available to engage in discussions about your code, running pipelines, writing configs, or anything else related to nf-core and Nextflow.
+They will hang out in Gather during designated hours every week, ready to provide guidance and insights or just chat about your favorite pipeline.
```

Comment:
```suggestion
They will hang out in Gather during designated hours every week, ready to provide guidance and insights or just chat about your favourite pipeline.
```
🇬🇧 
(impressed by your restraint to not give an example for a favorite pipeline 😉 )